os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -851,13 +894,54 @@ _timerHandle _tizenrt_timerCreate(const signed char *pcTimerName, osdepTickType
 	timer->timevalue = xTimerPeriodInTicks;
 	timer->data = pvTimerID;
 	timer->function = pxCallbackFunction;
+
+	if (!_tizenrt_timer_table_init) {
```

Comment:
_tizenrt_timer_table_init is shared variable. it seems it needs protection.

Reply:
_tizenrt_timer_table_init will only be used once, no other place will alter/modify it.
Does it need protection? if yes, is save_and_cli()(which is irqsave()) good enough?

Reply:
For example two tasks can call INIT_LIST_HEAD(). If you think it's ok then you don't need to protect it.

Reply:
It is better to have the protection, it is added now.